src/components/Button/Button.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react'
 import Button from 'react-bootstrap/Button'
 function Submit(props) {
-    const {title='Add'} = props;
+    const {title='Add',addHandler} = props;
     return (
         <div>
-            <Button variant="success" size="sm" className="mt-2">{title}</Button>
+            <Button variant="success" size="sm" className="mt-2" onClick={props.addHandler}>{title}</Button>
```

Comment:
variant="success" size="sm" these values should be dynamic via props not hardcoded